
---
uint8 RC_SW_UP = 0
uint8 RC_SW_MIDDLE = 1
uint8 RC_SW_DOWN = 2

float32[8] axes
int32[8] buttons